jobs/build/advisories/Jenkinsfile

```diff
@@ -0,0 +1,212 @@
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "vlaad@redhat.com"
+                        ],
+                        [
+                            name: 'PACKAGE_OWNER',
+                            description: "Owner's email address",
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "${buildUserEmail}"
+                        ],
+                        [
+                            name: 'IMPETUS',
+                            description: 'For which reason are those advisories being created',
+                            $class: 'hudson.model.ChoiceParameterDefinition',
+                            choices: [
+                                "standard",
+                                "cve",
+                                "ga",
+                                "test",
+                            ].join("\n"),
+                        ],
+                        [
+                            name: 'DATE',
+                            description: 'Format: YYYY-Mon-dd (i.e.: 2050-Jan-01)',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "${dateFormat.format(date)}"
+                        ],
+                        [
+                            name: 'DRY_RUN',
+                            description: 'Take no action, just echo what the job would have done.',
+                            $class: 'hudson.model.BooleanParameterDefinition',
+                            defaultValue: true
+                        ],
+                        commonlib.mockParam(),
+                        commonlib.suppressEmailParam(),
+                        [
+                            name: 'MAIL_LIST_SUCCESS',
+                            description: '(Optional) Success Mailing List\naos-cicd@redhat.com,aos-qe@redhat.com',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "",
+                        ],
+                        [
+                            name: 'MAIL_LIST_FAILURE',
+                            description: 'Failure Mailing List',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: [
+                                'aos-team-art@redhat.com'
+                            ].join(',')
+                        ],
+                        [
+                            name: 'SPECIAL_NOTES',
+                            description: '(Optional) special notes to include in the email',
+                            $class: 'hudson.model.TextParameterDefinition',
+                            defaultValue: ""
+                        ],
+                    ]
+                ],
+            ]
+        )
+
+        commonlib.checkMock()
+
+        currentBuild.description = ""
```

Comment:
oh yeah, one more thing... can `currentBuild.displayName` be something like `3.11.666 advisories [DRY RUN]`?

Reply:
I got this partially done ... since I stopped using raw bugzilla to create the placeholder bug and started using the new elliott command, I no longer need to ask for the `z` in version `x.y.z`. So for now, `currentBuild.displayName` is `3.11.z advisories [DRY RUN]`.

Is there any easy way to discover the `z` ?

Reply:
For 3.y.z we won't know for certain the `z` but you can look it up `brew latest-build rhaos-3.<y>-rhel-7-candidate atomic-openshift` and it will be the most recent known at the time (and possibly what we release).
I'd also be happy with just `3.11.z` - this is fiddly stuff